src/models/ids.rs

```diff
@@ -84,12 +84,27 @@
     };
 }
 
-from_base62id! {
-    ModId, ModId;
-    UserId, UserId;
-    VersionId, VersionId;
-    TeamId, TeamId;
+macro_rules! impl_base62_display {
+    ($struct:ty) => {
+        impl std::fmt::Display for $struct {
+            fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                f.write_str(&base62_impl::to_base62(self.0))
+            }
+        }
+    };
+}
+impl_base62_display!(Base62Id);
+
```

Comment:
Concern

Reply:
If you want to write out the exact same code for every `Id` struct, go ahead.  I made this as a quick patch for Geometrically so that `Display` would work for the ids.